common/scala/src/main/resources/application.conf

```diff
@@ -82,4 +82,9 @@ whisk {
         activations-ddoc = "whisks.v2"
         activations-filter-ddoc = "whisks-filters.v2"
     }
+    # transaction ID related configuration
+    transactions {
+        stride = 1
+        stride = ${?CONTROLLER_INSTANCES}
```

Comment:
since this is in the common conf, it feels wrong to have this tied to controller instances.
` +        stride = ${?CONTROLLER_INSTANCES} `?